### ISSUE TITLE
Events and props are not getting handled correctly in UMD components

### ISSUE THREAD

**Issue by prdatur** [quasarframework/quasar#16428]
### What happened?

I need to use the umd version to include vue + quasar within an existing project.
I created a ui-component with quasar-cli and created a small test component with a q-timepicker component.

First Issue:
When I press-down on a number and want to drag it to another number it directly selects the hour, this also happens on the non-umd version, but when I click on the current hour, to re-reselect the hour, on the non-umd version i can drag the clock pin to the new hour, within the UMD version it auto selects directly when i hover over another hour-number.

Second Issue:
I have provided minuteOptions prop with a computed value, which generates an array `[0, 15, 30, 45]` which works in the non-umd version but no options are available within the UMD version.

br,
PrDatur

### What did you expect to happen?

The clock slider should be draggable and the minute options should be available (0, 15, 30, 45)

### Reproduction URL

https://codepen.io/dev-ackc/pen/KKbrPBz

### How to reproduce?

First Issue reproduce:
- Do a fresh page reload
- Mouse down on hour 16, while holding mouse down, the hour get's directly selected
- Click on the current hour in the header to re-select the hour
- mouse down on hour 16, hold mouse down => nothing happens which is correct
- move mouse to hour 15 = after the mouse leaves the hour 16 and enters hour 15 it get's directly selected without the mouse up event

Second issue:
- Select any hour,
- no minute options are available to click

### Flavour

UMD

### Areas

Components (quasar)

### Platforms/Browsers

Tested on firefox and chrome

### Quasar info output

```Shell
Operating System - Windows_NT(10.0.19045) - win32/x64
NodeJs - 18.14.2

Global packages
  NPM - 9.5.0
  yarn - 1.22.19
  @quasar/cli - 2.3.0
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.12.7 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.6.2 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.16.7 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.3.4 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.2.5
  pinia - Not installed
  vuex - Not installed
  vite - 2.9.16 -- Native-ESM powered web dev build tool
  eslint - 8.50.0 -- An AST-based pattern checker for JavaScript.
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*
```

**Comment by prdatur** [comment]
Ok figured it out, that this behaviour exists if the time isn't filled out.
Issue 2 has been resolved, there is an error with tsconfig and the helper function __spreadArray, 
Issue 1 still exists if there isn't a valid time, valid time also means that the minutes must be one of the valid minuteOptions in order to correctly drag hours / minutes in first clicks.

Should this issue left open, to fix "empty dates" ?

**Comment by rstoenescu** [comment]
Thanks for reporting!
Small note: this bug is not UMD specific.

Fix will be available in Quasar v2.12.8
